lib/plugins/aws/deploy/lib/initializeResources.js

```diff
@@ -45,17 +44,13 @@ module.exports = {
         .Statement.concat(this.serverless.service.provider.iamRoleStatements);
     }
 
+    this.serverless.service.custom = {};
```

Comment:
Yep. Good catch. The `custom` object is reserved for plugin authors so that plugin users can setup specific plugin related configuration.

a `this.custom` could be a good solution for this problem.